### PR TITLE
Create tables

### PR DESCRIPTION
**Pull request by dneff1895**
Created files for the different tables.

### CODE REVIEW

File: SQL/Create-Table_dbo.amar_Tailoring.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE dbo.amar_tailoring (
+	[UID] INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
```
Comment (albertharrell) [inline SQL/Create-Table_dbo.amar_Tailoring.sql:2]: Can you fix spacing here

File: SQL/Create-Table_dbo.amar_fishing.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
+CREATE TABLE dbo.amar_fishing
+(
+    [UID] INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    ActionName VARCHAR(150) NOT NULL,
+    ActionNameDrop1 VARCHAR(150)
+        ActionNameDrop1QuantityMin INT,
```
Comment (albertharrell) [inline SQL/Create-Table_dbo.amar_fishing.sql]: Can you fix the spacing on this file too please so they are inline

File: SQL/Create-Table_dbo.amar_mining.sql [hunk 3]
```diff
@@ -0,0 +1,33 @@
+CREATE TABLE dbo.amar_mining
+(
+    [UID] INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+    ActionName VARCHAR(150) NOT NULL,
+    ActionNameDrop1 VARCHAR(150)
+        ActionNameDrop1QuantityMin INT,
```
Comment (albertharrell) [inline SQL/Create-Table_dbo.amar_mining.sql]: Can you fix line spacing in this file

File: SQL/Create-Table_dbo.amar_smithing.sql [hunk 4]
```diff
@@ -0,0 +1,18 @@
+CREATE TABLE dbo.amar_smithing (
+	[UID] INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
```
Comment (albertharrell) [inline SQL/Create-Table_dbo.amar_smithing.sql:2]: can you fix the line spacing

**Review by albertharrell — CHANGES_REQUESTED** [review]

File: SQL/Create-Table_dbo.amar_Tailoring.sql (hunk 1 above)
Comment (dneff1895) [reply SQL/Create-Table_dbo.amar_Tailoring.sql:2]: Corrected the spacing.

File: SQL/Create-Table_dbo.amar_fishing.sql (hunk 2 above)
Comment (dneff1895) [reply SQL/Create-Table_dbo.amar_fishing.sql]: Corrected the spacing on here as well.

File: SQL/Create-Table_dbo.amar_mining.sql (hunk 3 above)
Comment (dneff1895) [reply SQL/Create-Table_dbo.amar_mining.sql]: Corrected Spacing here.

File: SQL/Create-Table_dbo.amar_smithing.sql (hunk 4 above)
Comment (dneff1895) [reply SQL/Create-Table_dbo.amar_smithing.sql:2]: Corrected Spacing here.

**Review by albertharrell — APPROVED** [review]

**Pull request merged**